PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
+
+from PhysicsTools.PatAlgos.tools.helpers import getPatAlgosToolsTask, addToProcessAndTask
+
+def applyDeepBtagging( process, postfix="" ) :
+
+    task = getPatAlgosToolsTask(process)
+
+    from PhysicsTools.PatAlgos.tools.jetTools import updateJetCollection
+
+    process.load('PhysicsTools.PatAlgos.slimming.slimmedJets_cfi')
+
+    # update slimmed jets to include DeepFlavour (keep same name)
+    # make clone for DeepFlavour-less slimmed jets, so output name is preserved
+    addToProcessAndTask('slimmedJetsNoDeepFlavour', process.slimmedJets.clone(), process, task)
+    updateJetCollection(
+       process,
+       jetSource = cms.InputTag('slimmedJetsNoDeepFlavour'),
+       # updateJetCollection defaults to MiniAOD inputs but
+       # here it is made explicit (as in training or MINIAOD redoing)
+       pvSource = cms.InputTag('offlineSlimmedPrimaryVertices'),
+       pfCandidates = cms.InputTag('packedPFCandidates'),
+       svSource = cms.InputTag('slimmedSecondaryVertices'),
+       muSource = cms.InputTag('slimmedMuons'),
+       elSource = cms.InputTag('slimmedElectrons'),
+       jetCorrections = ('AK4PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'None'),
+       btagDiscriminators = [
+          'pfDeepFlavourJetTags:probb',
+          'pfDeepFlavourJetTags:probbb',
+          'pfDeepFlavourJetTags:problepb',
+          'pfDeepFlavourJetTags:probc',
+          'pfDeepFlavourJetTags:probuds',
+          'pfDeepFlavourJetTags:probg',
+       ],
+       postfix = 'SlimmedDeepFlavour'+postfix,
+       printWarning = False
+    )
+
+    # slimmedJets with DeepFlavour (remove DeepFlavour-less)
+    delattr(process, 'slimmedJets')
+    addToProcessAndTask('slimmedJets', getattr(process,'selectedUpdatedPatJetsSlimmedDeepFlavour'+postfix).clone(), process, task)
+    # delete module not used anymore (slimmedJets substitutes)
+    delattr(process, 'selectedUpdatedPatJetsSlimmedDeepFlavour'+postfix)
+
+
+
+    ## update slimmed jets to include DeepFlavour (keep same name)
```

Comment:
I think that here it's more practical to fold this with an if
```python
 #switch to True if needed for private tests
enableDoubleB = False
if enableDoubleB:
...
```

Reply:
done in eb073bb2b978713a0244a5c2e62cd49ded51c346